src/storage/secondary/block/rle_block_builder.rs

```diff
@@ -88,9 +163,14 @@ where
         }
         self.rle_counts.push(self.cur_count);
         encoded_data.put_u32_le(self.rle_counts.len() as u32);
+        let offset = encoded_data.len();
+        encoded_data.put_u32_le(0xffffffff); // placeholder
```

Comment:
Do we really need this length? We can read until the end when decoding given the number of RLE count item.